impl_core/src/main/java/io/opencensus/implcore/trace/propagation/BinaryFormatImpl.java

```diff
@@ -18,7 +18,7 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
-import io.opencensus.internal.DefaultVisibilityForTesting;
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
I thought we would eventually remove guava from api and impl?

Reply:
Yes, we'll probably remove Guava from the implementation if everyone agrees in #1113, but I don't think we've started removing any uses yet.  I think that finding a way to replace all uses of common utilities, like VisibleForTesting, that works across libraries should be a separate change.  I'll open an issue for that.

Reply:
I opened #1138.